pom.xml

```diff
@@ -85,6 +85,7 @@
     <deb.dependency.list>${deb.python.ver}</deb.dependency.list>
     <solr.version>7.5.0</solr.version>
     <hadoop.version>3.1.1</hadoop.version>
+    <hdp-hadoop.version>3.1.1.3.0.2.0-50</hdp-hadoop.version>
```

Comment:
Will this need to be updated periodically like some metrics dependency versions?

Reply:
@adoroszlai that is a released version from HDP repos. if 3.2.0 will go out to apache release, that will be used.